### DISCUSSION TITLE
windows folder permissions change issue

### DISCUSSION THREAD

**Discussion by Noldorinn** [AdguardTeam/AdGuardHome#7426] [Q&A]
Hi, i used on windows 10 64 bit. After a while, I saw that the adguard service could not be started in services. Then I went to the folder location (C:\AdGuardHome) and noticed that the folder permissions had changed. When I tried to change the permissions, I could not change the permissions of AdGuardHome.exe and AdGuardHome.yaml. When I try to look at the advanced security settings of AdGuardHome.exe, I get the warning “you are not authorized to view the security properties of this object, even if you are the administration user.” How can I solve this problem?
![Ekran Alıntısı](https://github.com/user-attachments/assets/08d7cf20-92db-4bd1-a9da-8d7be9f72587)

**Comment by vfxraven19** [comment]
I'm having this same issue too, started happening with this new version. The folder permissions get screwed up.

**Reply by jaypatrick** [reply]
Having the same issue, and now I can't even browse the administration page. Doesn't matter if I reinstall and reconfigure and use the same .yaml config, eventually the permissions get locked down, and then you cannot browse the admin pages (are those ports getting locked due to the prior version so that a new installation can't bind?)

I cannot stress enough how much havoc this is wreaking on my clients' networks, especially domains. I have a lot of pissed of domain admins right now who are very unhappy with this change, b/c they are ALL locked out, even though of course they are members of the local administrators group.

This is an extremely serious issue, and the fact that adguard has not commented on this issue is making me wonder if they are taking us seriously. You cannot just change permissions on a folder like this, this change was obviously not vetted nor thought throughby adguard. You want to piss off sysadmins, change permissons on folders on them.

If this isn't fixed ASAP, I'm advising my clients to move back to NextDNS. That's how serious this problem is.

**Reply by DanFunky** [reply]
Same here. Back to the old version and the issue does not occur. I upgraded again and seeing the problem again. Permissions in Windows get set to Administrator Write only on the AdGuardHome folder with no inherited permissions. Only fix is to take ownership of the entire folder structure and replace permissions so the service can start.

**Reply by McR1983** [reply]
Hi, which old version is OK?

**Reply by Xenfernal** [reply]
> Hi, which old version is OK?

Stable [v0.107.53](https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.107.53) or older. The newest beta release ([v0.108.0-b.61](https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.108.0-b.61)) should not give these permission errors too but if you do get them, you can use --no-permcheck parameter while installing the service again using commandline.

Edit: Looks like [v0.107.55](https://github.com/AdguardTeam/AdGuardHome/releases/tag/v0.107.55) is out now which has the same fixes as in beta as well. Works fine for me at the moment.

**Reply by McR1983** [reply]
Thanks i have installed the Beta it works!
